modules/ppcp-onboarding/assets/js/onboarding.js

```diff
@@ -247,6 +247,12 @@ function ppcp_onboarding_productionCallback(...args) {
             }
         );
 
+        if (sandboxSwitchElement.checked) {
+            sandboxSwitchElement.checked = false;
+        } else {
+            sandboxSwitchElement.checked = true;
+        }
```

Comment:
Seems we are switching the checkbox status here, if so maybe just: 
`sandboxSwitchElement.checked = ! sandboxSwitchElement.checked;`

Reply:
d7aef159f954a65ebd2b720e2d98abaaed971653 done